fix some things in xfce4

Yisus7u7 opened the pull request.

Grimler91 commented on packages/xfce4/build.sh:
```diff
@@ -1,9 +1,9 @@
-TERMUX_PKG_VERSION=2.1
-TERMUX_PKG_REVISION=16
+TERMUX_PKG_VERSION=4.16
+TERMUX_PKG_REVISION=17
```
The TERMUX_PKG_REVISION line can be removed since TERMUX_PKG_VERSION is increased

Yisus7u7 replied:
ok, I'll keep making more changes and remove the draft when I'm done.

Grimler91 reviewed (changes requested):
Nice, informative, commit messages, thanks!

Yisus7u7 commented:
@Grimler91 

there is an error in x86_64 to download lxde-icon-theme ...

Yisus7u7 commented:
I'm done, these are the changes:

- Fixed incorrect information in xfce4 metapackage

- added the xfce4-screensaver package and added to the xfce4-goodies meta package, also galculator was added to xfce4-goodies.

Fixed issues

- Before when changing themes, icons and cursor appearance in xfce4, the application crashed or crashed, the gsettings-desktop-schemas package solves that.

- Fixed glib error in xfce4 `glib error: gnome desktop interface is not installed`

Yisus7u7 commented:
> Nice, informative, commit messages, thanks!

Thank you very much, I have already added many packages to x11-repo and decided now to be more organized.

Yisus7u7 commented:
Over time I will add new apps and things, my next goal is to add the gnuplot package, and then some apps and terminal emulators.

xeffyr commented on packages/xfce4/build.sh:
```diff
@@ -1,8 +1,8 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/termux/x11-packages
+TERMUX_PKG_HOMEPAGE=https://xfce.org/
 TERMUX_PKG_DESCRIPTION="A metapackage that installs all parts of the XFCE desktop environment"
 TERMUX_PKG_LICENSE="Public Domain"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=2.1
+TERMUX_PKG_VERSION=4.16
 TERMUX_PKG_REVISION=16
 TERMUX_PKG_SKIP_SRC_EXTRACT=true
 TERMUX_PKG_DEPENDS="thunar, xfce4-panel, xfce4-session, xfce4-settings, xfconf, xfwm4, xfce4-notifyd"
```
This package is managed by @termux and is not part of the original XFCE project. Please don't touch version and home page.

Yisus7u7 replied:
> This package is managed by @termux and is not part of the original XFCE project. Please don't touch version and home page.

I still highly recommend putting the original version of xfce4 (4.16)

xeffyr commented:
> my next goal is to add the gnuplot package

It is in main repo and is already built with X11 support.

Yisus7u7 commented:
> > my next goal is to add the gnuplot package
> 
> It is in main repo and is already built with X11 support.

So I will add a "gnuplot-qt5" package which will bring gnuplot with Qt widget support, right?

Yisus7u7 commented:
@xeffyr 

voila, i reverted the changes in xfce4, you can now merge them.

Yisus7u7 commented:
oh i can't believe i misspelled that i need some coffee 😂
